cmd/nvidia-ctk/cdi/generate/generate.go

```diff
@@ -183,17 +183,6 @@ func (m command) validateFlags(c *cli.Context, opts *options) error {
 		return fmt.Errorf("invalid output format: %v", opts.format)
 	}
 
-	opts.mode = strings.ToLower(opts.mode)
```

Comment:
This was removed because I didn't want to maintain the list in two places. Let me pull in some changes that I have in another commit to improve things